craft_parts/plugins/cmake_plugin.py

```diff
@@ -0,0 +1,125 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2020 Canonical Ltd.
```

Comment:
Since we're porting this to craft-parts we should use 2020-2022 in the copyright notice.